src/usecases/export_calendar/FileStorage.java

```diff
@@ -0,0 +1,22 @@
+package usecases.export_calendar;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class FileStorage {
+    public static String saveToFile(String calendar, String name) throws IOException {
+
+        String downloadsPath = "Downloads/";
```

Comment:
Just something to think about: Is Downloads the most descriptive folder name here? Technically, we are not downloading anything, we are simply exporting our calendar from the UI to a .ics file.

As such, you could possibly rename this folder to something more descriptive, such as CalendarExports.

Reply:
Obviously, if you do choose to go through with this name change, you would have to update your tests with the new chosen name.

Reply:
I'll make the change. Originally, I wanted to export to the Downloads file on the computer but changed it. Thanks for the catch.